### PR TITLE
Add an external CSS file to a JavaFX Application

### LINKED ISSUE #3734: Add an external CSS file to a JavaFX Application
### Proposed title of the article

Add an external CSS file to a JavaFX Application

### Introduction paragraph (2-3 paragraphs):

Are you having trouble attaching an external CSS file to a JavaFX application? Having issues on how to link an externally designed styling sheet to your program? Worry no more! In this article, you will find answers to these problems.

By now, as you have noticed that JavaFX is easy to work with. Challenges come when one decides to style the JavaFX application. It's a cumbersome procedure since each element requires its style. Issues of the designer remembering previously used styles on different elements arises again and again. This is pretty challenging. In the FXML file, it also is hard to follow up with the styling in the code since it is not continuous. You may also have noticed that the keywords used during styling are different from those of an HTML file. Well, there is a solution to this problem.

### Key takeaways: 

At the end of the article, you will gain the following knowledge:

- Saving a repetitive style for easier use on other elements
- Adding an external stylesheet to the application
- Referencing elements in the stylesheet by adding JavaFX classes
- Linking the stylesheet to multiple scenes

### References: 

- N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by blacklihgt**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #3734

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Review by Dawe-7 — APPROVED** [review]
@blacklihgt, kindly go through the changes made in the commits to improve in your future articles.

@Kayere, this article is ready for final review. Thanks

**Comment by LinusMuema** [comment]
Hi @blacklihgt 

Please provide a better conclusion giving readers more insights on what has been covered, what they can do to improve, further research, etc. Give a better conclusion. 

Also, I found it particularly hard to follow through the article as the grammar had a lot of issues and readability was poor. Please check on the changes I have made in that commit before we publish the article and ping me when done.

cc: @Dawe-7 please check on readability in the articles before a final review as they may still be closed due to the same.

**Comment by blacklihgt** [comment]
@LinusMuema @Dawe-7 added the conclusion and did some more grammar checks and corrections
Please confirm it

**Comment by Dawe-7** [comment]
@LinusMuema, The author has committed some changes as requested and they look good to me. Could we have a final review of this PR, thanks.

**Comment by Dawe-7** [comment]
@blacklihgt, looking back at your last commit, i can see you missed out on the important thing that was requested by @LinusMuema . Kindly add a better conclusion to your tutorial as was requested to prevent closure of your PR.

**Comment by hectorkambow** [comment]
@blacklihgt 
Please be sure to go through the edits made by your peer reviewer and content moderator to improve future articles.
As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published.

Our team of peer reviewers do not provide revisions services at this time in order to review them all in a timely manner. 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 

Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

ANY FUTURE ARTICLE SUBMITTED WITH GLARING ERRORS (that requires many edits) WILL BE IMMEDIATELY CLOSED.

cc @LinusMuema & @Dawe-7  

🚀

**Comment by blacklihgt** [comment]
> @LinusMuema, The author has committed some changes as requested and they look good to me. Could we have a final review of this PR, thanks.

@Dawe-7 @LinusMuema A better Conclusion has been added, please check it out and ping me for any inquiries

**Comment by LinusMuema** [comment]
Hi @blacklihgt 

You can check the changes I made for future articles to improve your writing.

cc: @Dawe-7
